src/app/organisation/exportusers/get.controller.js

```diff
@@ -25,9 +24,11 @@ module.exports = (req, res) => {
     logger.debug('In organisation export users get controller');
     const cookie = new CookieModel(req);
 
-    orgApi.getUsers(cookie.getOrganisationId())
+    // to retrieve all rows, the 1st to the last row from the users table
+    // set the offset up to the end of the result set (a large number for the second parameter).
+    orgApi.getUsers(cookie.getOrganisationId(), 1, 999999999999999)
```

Comment:
Are you retrieving all the users at this line?

Reply:
yes, all the users based on the organisation